src/openms_gui/include/OpenMS/VISUAL/APPLICATIONS/TOPPViewBase.h

```diff
@@ -300,6 +300,8 @@ public slots:
     void annotateWithAMS();
     /// Annotates current layer with ID data
     void annotateWithID();
+    /// Switch to Top-Down view
+    void topDown();
```

Comment:
the name is a bit too generic.
What exactly does it do? Show charge ladders? Or more?